cloud: update public endpoint firewall steps (#22807)
pingcap/docs: tidb-cloud/configure-serverless-firewall-rules-for-public-endpoints.md

2. In the left navigation pane, click **Settings** > **Networking**.

3. On the **Networking** page, enable **Public Endpoint** if it is disabled.

> 4. (Optional) For a newly created {{{ .starter }}} or {{{ .essential }}} instance, TiDB Cloud enables **Allow_all_public_connections** by default. To allow access only from specified IP addresses or IP address ranges, click **...** in the row of **Allow_all_public_connections**, and then click **Delete**.

Reviewer: ```suggestion
4. (Optional) For a newly created {{{ .starter }}} or {{{ .essential }}} instance, TiDB Cloud enables **Allow_all_public_connections** by default. To restrict access to specific IP addresses or ranges, click **...** in the row of **Allow_all_public_connections**, and then click **Delete**.
```

> 5. In **Authorized Networks**, click **Add rule**, and then add the IP address or IP address range that you want to allow.

Reviewer: ```suggestion
5. In the **Authorized Networks** section, click **Add rule**, and then add the IP address or IP address range that you want to allow.
```

    - To add the current IP address of your computer, click **Add Current IP**. This automatically creates a firewall rule with the public IP address of your computer, as perceived by TiDB Cloud.

        > **Note:**
        >

>         > In some situations, the IP address perceived by the TiDB Cloud console might be different from the IP address used by your database client to connect to TiDB Cloud. Therefore, you might need to change the start and end IP addresses to make the rule work as expected. You can use a search engine or an online tool to check your public IP address. For example, search for "what is my IP".

Reviewer: ```suggestion
        > The IP address perceived by the TiDB Cloud console might differ from the IP address used by your database client. If the rule does not work as expected, adjust the start and end IP addresses. You can use a search engine or an online tool to check your public IP address. For example, search for "what is my IP".
```

>     - To enable access from all AWS IP addresses if your {{{ .starter }}} or Essential instance is hosted on AWS, click **Add AWS Access**. This automatically creates a firewall rule for all AWS IP addresses. TiDB Cloud uses the reserved IP address **169.254.65.87** to represent all AWS IP addresses and regularly updates this list according to the official [AWS IP address list](https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html).

Reviewer: ```suggestion
    - To enable access from all AWS IP addresses if your {{{ .starter }}} or Essential instance is hosted on AWS, click **Add AWS Access**. This automatically creates a firewall rule that includes all AWS IP ranges. TiDB Cloud uses the reserved IP address **169.254.65.87** to represent AWS IP ranges and updates the list regularly based on the official [AWS IP address list](https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html).
```

    - To add more address ranges, specify a single IP address or a range of IP addresses. To limit the rule to a single IP address, enter the same IP address in the **Start IP Address** and **End IP Address** fields.

>         >**Note:**

Reviewer: ```suggestion
        > **Note:**
```
